tephra-core/src/test/java/co/cask/tephra/distributed/PooledClientProviderTest.java

```diff
@@ -158,15 +162,19 @@ private void startClientAndTestPool(Configuration conf) throws Exception {
   private static class RetrieveClient implements Callable<Integer> {
     private final PooledClientProvider pool;
     private final long holdClientMs;
+    private final CountDownLatch begin;
 
-    public RetrieveClient(PooledClientProvider pool, long holdClientMs) {
+    public RetrieveClient(PooledClientProvider pool, long holdClientMs,
+                          CountDownLatch begin) {
       this.pool = pool;
       this.holdClientMs = holdClientMs;
+      this.begin = begin;
     }
 
     @Override
     public Integer call() throws Exception {
       try (CloseableThriftClient client = pool.getCloseableClient()) {
+        begin.await();
```

Comment:
Since this latch is called `begin`, it makes sense to put this as the first statement of the `call()` method, right?